cli/cppcheckexecutor.cpp

```diff
@@ -240,7 +240,7 @@ namespace {
 
         void printRaw(const std::string &message) override
         {
-            std::cout << message << std::endl;
+            std::cout << message << std::endl; // TODO: should not append newline
```

Comment:
It is a bit of a misnomer right now since it is not `raw` (it has been an awkward name to begin with) as it adds to it.